### PR TITLE
Add v1.17.0 release blog post

### PR DESCRIPTION
**Pull request by pmuens**
**Todos:**

- [x] Write content for each headline / feature
- [x] Update contributors list

### CODE REVIEW

File: posts/2017-07-05-serverless-v1.17.0.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+serverless deploy
+
+serverless invoke --function hello
+```
+
+The F# service template supports all the configs and event sources you can find in [our AWS documentation](https://serverless.com/framework/docs/providers/aws/).
+
+### Skip deployment if files not changed
+
+Starting now, Serverless will automatically compare you current services files on disk to the remotely uploaded service files of your last deployment.
```
Comment (ryanmurakami) [inline posts/2017-07-05-serverless-v1.17.0.md]: 'services files' -> 'service files'

Comment (pmuens) [inline posts/2017-07-05-serverless-v1.17.0.md]: Nice catch 👍

**Comment by pmuens** [comment]
@worldsoup just pushed the last changes to the blog post. Should be ready for publishing release.

@ryanmurakami thanks again for the review! Just ping here once the video is done and we'll embed it ASAP.

File: posts/2017-07-05-serverless-v1.17.0.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
+---
+title: Serverless v1.17 - F# service template, deployment skipping when files not changed added
+description: F# service template, deployment skipping when files not changed and more in the Serverless Framework v1.17 release.
+date: 2017-07-05
+layout: Post
+thumbnail: https://s3-us-west-2.amazonaws.com/assets.blog.serverless.com/framework-v117.png
```
Comment (j0k3r) [inline posts/2017-07-05-serverless-v1.17.0.md]: The image doesn't work.
Will it be uploaded later?

Comment (pmuens) [inline posts/2017-07-05-serverless-v1.17.0.md]: Nice catch @j0k3r 👍 Thanks for reviewing!

This is just a placeholder right now. @worldsoup will update it before publishing.

**Comment by worldsoup** [comment]
schedule(07/06/2017 6:00 am)

**Comment by DavidWells** [comment]
✅ Post scheduled for **07/06/2017 at 6:00am (America/Los_Angeles)**

**To unscheduled the post:**

1. **[Delete the previous comment ⬆](https://github.com/serverless/blog/pull/215#issuecomment-313254628)** with schedule(time) information by clicking the ❌
  ```
  ┌──────────────────────────────────────────────┐
  │                                           X  │
  ├───────────────────────────────────────────▲──┤
  │                                           │  │
  │  schedule(mm/dd/yyyy h:mm A)   ┌────────┐ │  │
  │                                │ Delete ├─┘  │
  │                                └────────┘    │
  └──────────────────────────────────────────────┘
  ```
2. (Optionally) Delete this comment to keep things tidy

> Powered by the ⚡️ [Serverless post scheduler](https://github.com/serverless/post-scheduler) | Made with ❤️  by [Serverless](https://serverless.com/)

**Comment by pmuens** [comment]
Just added @ryanmurakami release video!

Thanks for that @ryanmurakami 🙏 🥇 👍

**Comment by DavidWells** [comment]
🎉 **The post has been published**

> Powered by the ⚡️ [Serverless post scheduler](https://github.com/serverless/post-scheduler) | Made with ❤️  by [Serverless](https://serverless.com/)

**Pull request closed without merging**